docs/sources/reference/components/beyla/beyla.ebpf.md

```diff
@@ -156,6 +166,40 @@ Name               | Type           | Description
 `owner_name`       | `string`       | Regular expression of Kubernetes owners of running Pods to match.                                           | `""`    | no
 `pod_labels`       | `map(string)`  | Key-value pairs of labels with keys matching Kubernetes Pods with the provided value as regular expression. |  `{}`   | no
 
+### prometheus block
+
+This block configures which metrics Beyla collects.
```

Comment:
I don't see a way of disabling this feature, is it ok? What if the user is only interested in traces?

Reply:
looking at the code, I don't think that this is the case. The Convert() method will apply the default values which ["application"] and ["*"].